### PR TITLE
INTMDB-347: [Terraform] cloud_backup is not being correctly imported - issue 768

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

[Terraform] cloud_backup is not being correctly imported - issue 768

Link to any related issue(s):
https://github.com/mongodb/terraform-provider-mongodbatlas/issues/768

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

## Further comments

INTMDB-347

### CODE REVIEW

**Review by themantissa — APPROVED** [review]
LGTM!

File: mongodbatlas/resource_mongodbatlas_cluster.go [hunk 1]
```diff
@@ -1010,6 +1010,10 @@ func resourceMongoDBAtlasClusterImportState(ctx context.Context, d *schema.Resou
 		log.Printf(errorClusterSetting, "name", u.ID, err)
 	}
 
+	if err := d.Set("cloud_backup", u.ProviderBackupEnabled); err != nil {
+		return nil, fmt.Errorf("couldn't import cluster %s in project %s, error: %s", *name, *projectID, err)
```
Comment (blva) [inline mongodbatlas/resource_mongodbatlas_cluster.go]: do we want to throw the same error as `Clusters.Get`? should probably use different wording here.

**Review by gssbzn — APPROVED** [review]
LGTM but some changes some unrelated to the bugfix

**Review by blva — CHANGES_REQUESTED** [review]

File: mongodbatlas/resource_mongodbatlas_cluster.go (hunk 1 above)
Comment (martinstibbe) [reply mongodbatlas/resource_mongodbatlas_cluster.go]: Added additional content relating error to backup configuration

**Review by blva — APPROVED** [review]
LGTM

**Pull request merged**